api-tests/server/platform_test.go

```diff
@@ -35,9 +35,10 @@ import (
 func TestPlatform(t *testing.T) {
 	client := platformClient.Default.Platform
 	serverClient := serverClient.Default.Server
+
+	const serverName string = "my PMM"
+	email, password := os.Getenv("PERCONA_TEST_PORTAL_USERNAME"), os.Getenv("PERCONA_TEST_PORTAL_PASSWORD")
 	t.Run("connect and disconnect", func(t *testing.T) {
-		const serverName string = "my PMM"
-		email, password := os.Getenv("PERCONA_TEST_PORTAL_EMAIL"), os.Getenv("PERCONA_TEST_PORTAL_PASSWORD")
```

Comment:
I'm confused about the `env` names PERCONA_TEST_PORTAL_USERNAME vs PERCONA_TEST_PORTAL_EMAIL, because `t.Skip` calls still mention the PERCONA_TEST_PORTAL_EMAIL, but in fact we use another variable named PERCONA_TEST_PORTAL_USERNAME.

If we should use the PERCONA_TEST_PORTAL_USERNAME indeed, then the t.Skip calls should also be updated

Reply:
i forgot to change other occurences, changed it now thanks